caf-logging-logback-converters/src/main/java/com/github/cafapi/logging/logback/converters/MaybeJsonMessageConverter.java

```diff
@@ -76,4 +90,27 @@ public String convert(final ILoggingEvent event)
             return sb.toString();
         }
     }
+    private String getStacktraceLineLimit(final List<String> optionList)
+    {
+        if (Objects.nonNull(optionList) && 1 < optionList.size()) {
+            final String s = optionList.get(1);
+            if (s.equals(HYPHEN)) {
+                return null;
+            }
+        }
+        return null;
+    }
+
+    private String getPackages(final List<String> optionList)
```

Comment:
Methods that can be `static` usually should be:

```suggestion
    private static String getPackages(final List<String> optionList)
```